.travis.yml

```diff
@@ -27,20 +27,6 @@ env:
     - DEPLOYMENT_PERL_VERSION=5.18
 
 
-before_install:
-  # We need to run a pre-1.0 instance of ES until we update everything.
-  - wget https://download.elasticsearch.org/elasticsearch/elasticsearch/elasticsearch-0.90.13.deb
-  - sudo dpkg -i --force-confdef elasticsearch-0.90.13.deb
-  - sudo service elasticsearch restart
-
```

Comment:
I think we can keep everything below this line.  It's just the Elasticsearch install we don't need to downgrade.